mentoring/gsoc/2024/project_ideas.md

```diff
@@ -26,3 +26,13 @@ If you are a Prometheus-Operator team member and consider mentoring during the G
 ---
 
 ## Ideas
+
+### `*Monitor` status subresource
+
+- Description: Currently, there are 4 Custom Resource Definitions (CRDs) fully dedicated to configure Prometheus scrape targets, `PodMonitor`, `ServiceMonitor`, `Probes` and `ScrapeConfig`, and all of them are missing the `.Status` subresource. This project is about implementing the status subresource for all 4 CRDs to increase visibility and debuggability of scrape configuration. The fields added to the status subresource will be discussed deeply during the mentorship, but a potential idea is to list the Prometheus/PrometheusAgent objects that selected each Monitor object and how many targets are being scraped.
+- Recommended Skills: Go, Kubernetes, Kubernetes CRDs.
+- Expected project size: medium
```

Comment:
I'd go with "large", like for the Daemonset topic.

Reply:
Yeah, I was a bit in doubt with this one, but let's be safe and classify as big